control_panel_api/management/commands/load_kube_users.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+from kubernetes import config, client
+
+from control_panel_api.models import User
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Import Kubernetes user-secrets data into the User.  Uses KUBECONFIG envvar or defaults to ~/.kube/config"
+
+    USER_SECRETS_NAME = 'user-secrets'
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+        config.load_kube_config()
+        v1 = client.CoreV1Api()
+        results = v1.list_secret_for_all_namespaces()
+
+        user_secrets = [item.data for item in results.items if item.metadata.name == Command.USER_SECRETS_NAME]
+
+        users = []
+        for item in user_secrets:
+            username = base64.b64decode(item['username'])
```

Comment:
Right, just checked the documentation of [`django.contrib.auth.models.User`](https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#user-model) and it seems it has separate fields for first name and last name (which makes sense), while we have the full name in the secret, not sure how to handle this (I mean, we may put the string before the first space in the `first_name` and the rest in `last_name` as a good guess)